proxy-robots-module/src/main/java/orwell/proxy/config/source/ConfigurationFile.java

```diff
@@ -0,0 +1,39 @@
+package orwell.proxy.config.source;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.config.ConfigModel;
+import orwell.proxy.config.Configuration;
+
+import javax.xml.bind.JAXBException;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+/**
+ * Created by Michaël Ludmann on 6/9/15.
+ */
+public class ConfigurationFile extends Configuration {
+    private final static Logger logback = LoggerFactory.getLogger(ConfigurationFile.class);
+    private final String filePath;
+
+    public ConfigurationFile(final String filePath) throws FileNotFoundException {
+        this.filePath = filePath;
+
+        final File file = new File(filePath);
+        if (!file.exists() || file.isDirectory()) {
+            throw new FileNotFoundException(filePath);
```

Comment:
We should probably throw a more meaningful exception and we could have a factory for the similar ones.

Something like
```java
throw new FileOrDirectoryExceptionsFactory.getFactory(File file).throwAdequateExceptionPlease()
```